driver/src/main.rs

```diff
@@ -97,11 +42,12 @@ fn main() {
 		let received = rx.recv().unwrap();
 		println!(": {}", received);
 
-		// initializing all needed web3 variables
 		let db_host = env::var("DB_HOST").unwrap();
-		let db_port = env::var("DB_PORT").unwrap();
-		let client = Client::connect(&db_host, db_port.parse::<u16>().unwrap())
-			.expect("Failed to initialize standalone client");
+        let db_port = env::var("DB_PORT").unwrap();
```

Comment:
Could you spend some time to fix your indentation settings in your IDE? I have seen this being messed up a lot in recent PRs.